services/Facebook.json

```diff
@@ -23,6 +23,47 @@
         "removeHelpButtons",
         "cleanUrls"
       ]
+    },
+    "devPaymentsTos": {
```

Comment:
In order to avoid inflating document types, I believe this would more accurately be represented by describing a `devPolicy` document for a separate service which could be called “Facebook Payments”.